packages/node_modules/@webex/react-component-md-choiceset-input/src/index.js

```diff
@@ -46,7 +46,12 @@ class MDChoiceSetInput extends AdaptiveCard.ChoiceSetInput {
 
       if (this.isCompact) {
         // Render Select box
-        let defaultOptionText = DEFAULT_VALUE_SELECT_BOX;
+        let defaultOptionText = null;
+
+        if (defaultChoice === null && this.choices.length > 0) {
+          defaultOptionText = this.choices[0].title;
+          this.value = this.choices[0].value;
```

Comment:
`this.value` is assigned but never used.

Reply:
`this.value` is a default variable